### PR TITLE
Update mapping

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
## Description

Update mapping:
* [x] Remove 'totally
* [x] Remove 'very'
* [x] Change 'various ' to 'multiple'
* [x] Remove 'various'
* [x] Change 'i.e.' tot 'that is'
* [x] Write shorter sentences

## Type of suggestion

* [x] Styleguide

## Product & version (if relevant)
Umbraco CMS v10, 12, 13 & 14

### CODE REVIEW

File: 10/umbraco-cms/reference/mapping.md [hunk 1]
```diff
@@ -1,28 +1,26 @@
...
 
 {% hint style="info" %}
-UmbracoMapper replaced AutoMapper which was an external dependency. AutoMapper builds the mapping code dynamically, based upon mapping profiles, which are defined as C# expressions. UmbracoMapper relies on static code, i.e. mappings need to be hand-written.
+UmbracoMapper was originally introduced to solve some issues in the Umbraco core code. However, it is fine for anyone to use in their custom site implementations or packages as they wish.
 
 This is not to be confused with the [UmbracoMapper package by Andy Butland](https://our.umbraco.com/packages/developer-tools/umbraco-mapper) of the same name.
 {% endhint %}
 
-UmbracoMapper was originally introduced to solve some issues in the Umbraco core code, however it is totally fine for anyone to use in their custom site implementations or packages as they wish.
+UmbracoMapper replaced AutoMapper which was an external dependency. AutoMapper builds the mapping code dynamically, based upon mapping profiles, which are defined as C# expressions. UmbracoMapper relies on static code, that is, mappings need to be hand-written.
```
Comment (alina-tincas) [inline 10/umbraco-cms/reference/mapping.md:15]: I suggest to leave back the sentence with **UmbracoMapper replaced AutoMapper**  in the hint note, as usually we list changes in hints note and in this case it makes sense to leave this in a hint note. Then with the sentence **UmbracoMapper was originally introduced** you can move this above the hint note or after the hint as it was💡

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Hi @erikjanwestendorp thank you for the PR 🙌 

All looks good except that somehow on v10 you have replaced 2 sentences with each other 🙈

File: 10/umbraco-cms/reference/mapping.md [hunk 2]
```diff
@@ -1,28 +1,26 @@
...
+Often in code there is a need to 'map' one object's properties to another type of object. The 'type of objects' are not related by inheritance or interface. (Think database layer object, passing information to a presentation layer ViewModel etc). In these circumstances, it can save time and provide consistency to consolidate the logic to map between the options into one set of 'Mapping' rules.
 
 {% hint style="info" %}
-UmbracoMapper replaced AutoMapper which was an external dependency. AutoMapper builds the mapping code dynamically, based upon mapping profiles, which are defined as C# expressions. UmbracoMapper relies on static code, i.e. mappings need to be hand-written.
+UmbracoMapper was originally introduced to solve some issues in the Umbraco core code. However, it is fine for anyone to use in their custom site implementations or packages as they wish.
```
Comment (alina-tincas) [inline 10/umbraco-cms/reference/mapping.md:10]:
```suggestion
UmbracoMapper replaced AutoMapper which was an external dependency. AutoMapper builds the mapping code dynamically, based upon mapping profiles, which are defined as C# expressions. UmbracoMapper relies on static code, that is, mappings need to be hand-written.
```

File: 10/umbraco-cms/reference/mapping.md (hunk 1 above)
Comment (alina-tincas) [inline 10/umbraco-cms/reference/mapping.md:15]:
```suggestion
UmbracoMapper was originally introduced to solve some issues in the Umbraco core code. However, it is fine for anyone to use in their custom site implementations or packages as they wish.
```

**Review by alina-tincas — COMMENTED** [review]
making those changes as it it only in 1 file

**Review by alina-tincas — APPROVED** [review]

**Comment by erikjanwestendorp** [comment]
@alina-tincas Sorry! I completely forgot about this one 😅

**Comment by alina-tincas** [comment]
No worries, I just noticed that I can also make the changes 🙈 🙌

**Pull request merged**